service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -121,10 +123,24 @@ class TestRunSetsApiController {
   private final UUID methodId = UUID.randomUUID();
   private final UUID methodVersionId = UUID.randomUUID();
   private final UUID dockstoreMethodVersionId = UUID.randomUUID();
-  private final String workflowUrl =
-      "https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/hello/hello.wdl";
+  private final String gitHubRepository = "cromwell";
+  private final String gitHubOrganization = "broadinstitute";
+  private final String gitHubBranchOrTag = "develop";
+  private final String gitHubPath = "centaur/src/main/resources/standardTestCases/hello/hello.wdl";
+  private final String gitHubWorkflowRawUrl =
+      "https://raw.githubusercontent.com/%s/%s/%s/%s"
+          .formatted(gitHubOrganization, gitHubRepository, gitHubBranchOrTag, gitHubPath);
+
+  private final String gitHubWorkflowOriginalUrl =
+      "https://github.com/%s/%s/%s/%s"
+          .formatted(gitHubOrganization, gitHubRepository, gitHubBranchOrTag, gitHubPath);
+  private final GithubMethodDetails githubMethodDetails =
+      new GithubMethodDetails(gitHubRepository, gitHubOrganization, gitHubPath, false, methodId);
   private final Boolean isCallCachingEnabled = false;
+
+  // TODO: this variable says dockstore but is actually github... bad?
```

Comment:
FWIW it should be perfectly ok for a dockstore method to resolve to a URL in github. Probably ought to be to a raw github url though 🤔